lib/stormpath-sdk/resource/base.rb

```diff
@@ -57,6 +57,10 @@ def non_printable_properties
       @non_printable_properties ||= []
       Array.new @non_printable_properties
     end
+
+    def has_status(name: :status, status_list: :default)
```

Comment:
This looks more like a check to me. So, you shouldn't be able to call `has_status` for a resource that doesn't even have a status.

When you do a create or an update and try to set the status to something not defined for that resource the server will return an error.